src/seedu/addressbook/data/person/Person.java

```diff
@@ -84,4 +96,6 @@ public String toString() {
         return getAsTextShowAll();
     }
 
+
```

Comment:
Try not to add redundant new lines. You can use some plugins on your IDE to check such things.